sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/ShowPartitionsSuite.scala

```diff
@@ -34,4 +34,18 @@ class ShowPartitionsSuite extends command.ShowPartitionsSuiteBase with CommandSu
         "SHOW PARTITIONS cannot run for a table which does not support partitioning"))
     }
   }
+
+  test("null and empty string as partition values") {
```

Comment:
Could you add `SPARK-33889: ` prefix please?

Reply:
Added